server/src/test/java/org/candlepin/resource/HypervisorResourceTest.java

```diff
@@ -310,4 +312,82 @@ public void checkInCreatesNoNewConsumerWhenCreateIsFalse() throws Exception {
     public void ensureBadRequestWhenNoMappingIsIncludedInRequest() {
         hypervisorResource.hypervisorUpdate(null, principal, "an-owner", false);
     }
+
+    @Test
+    public void ensureEmptyHypervisorIdsAreIgnored() throws Exception {
+        Owner owner = new Owner("admin");
+
+        Map<String, ArrayList<GuestId>> hostGuestMap = new HashMap<String, ArrayList<GuestId>>();
+        hostGuestMap.put("", new ArrayList(Arrays.asList(new GuestId("GUEST_A"),
+                                         new GuestId("GUEST_B"))));
+        hostGuestMap.put("HYPERVISOR_A", new ArrayList(Arrays.asList(new GuestId("GUEST_C"),
+                                                     new GuestId("GUEST_D"))));
```

Comment:
The indentation is inconsistent here.  Just add one additional indention on line continuations instead of spacing all the way over past the containing parenthesis.